evm/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #![allow(clippy::too_many_arguments)]
 #![allow(clippy::type_complexity)]
 #![feature(generic_const_exprs)]
+#![allow(dead_code)]
```

Comment:
I don't think we should had that lint globally. Maybe add it locally to unused functions along with a todo to remove it once used.

Reply:
Oh right — I think that was for debugging and I forgot to remove it